web/tests/lynxkite.ts

```diff
@@ -1252,50 +1287,3 @@ function submitInlineInput(element, text) {
   safeSelectAndSendKeys(inputBox, text);
   okButton.click();
 }
-
-// A matcher for lists of objects that ignores fields not present in the reference.
```

Comment:
Oh, it really was here! Sorry I put it at the top. Could you please move it to the bottom? Then we would see the diff.

Also, this original had a nice juicy comment. Please keep that. Thanks!